markdown/summary.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+output: 
+  word_document:
+    reference_docx: "summary-template.docx"
+---
+
+```{r, echo=FALSE, message=FALSE, warning=FALSE,error=FALSE}
+
+# Create contact information
+contact1 <- c("**Federico Centoni**", "Information Analyst", 
+              "0131 314 1532", "federico.centoni@nhs.net")
+contact2 <- c("**Alice Byers**", "Senior Information Analyst", 
+              "0131 314 1725", "alice.byers@nhs.net")
+contact3 <- c("**Aghimien Iyayi-Igbinovia**", "Principal Information Analyst", 
+              "0141 207 1616", "aghimien.iyayi-igbinovia@nhs.net")
+
+contact <- data.frame(contact1, contact2, contact3)
+```
+
+# Percentage of End of Life Spent at Home or <br> 
+# in a Community Setting (Provisional)
+## Financial years ending 31 March 2011 to 2019
+#### An Official Statistics publication for Scotland
+
+#### Publication date: 28 May 2019
+##### RESTRICTED STATISTICS: embargoed to 09:30 29/05/2019
+
+### About this release
+
+This release by the Information Services Division provides information on the percentage of time spent at home or in a community setting in the last six months of life. This is one of the measures used to support and determine progress towards the Scottish Government’s [“Strategic Framework for Action on Palliative and End of Life Care”](http://www.gov.scot/Resource/0049/00491388.pdf). Data are presented by financial year from 2010/11 to 2018/19, by NHS Board, Health and Social Care Partnership of residence and for Scotland. 
+
+Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures. Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates.
+
+
+### Main Points
+
+* In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded.  For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
+
+* Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+
+**Chart here**
```

Comment:
I found you've got a superscript in the chart title. I tried to set it up by using:

ggtitle(expression(paste("Percentage of last six months of life spent at home or in a community setting; 2010/11 - ", "2018/19"^1)))

It worked but strangely it will drop the style set up in plot.title (e.g. not bold anymore). We need to look into it when creating the actual chart. Maybe need to use a normal heading instead of title in ggplot2.